src/Core/Domain/Product/Command/AssignProductToCategoryCommand.php

```diff
@@ -24,8 +24,6 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-declare(strict_types=1);
```

Comment:
For reviewers: adding this `declare(strict_types=1);` was a BC break because file existed without `declare(strict_types=1);` in 1.7.7